data/association-flag-map.yaml

```diff
@@ -79,18 +79,15 @@ columns:
     - name: slot_Shape_flag
       bit: 23
       doc: General Failure Flag
-    - name: slot_Shape_flag_unweightedBad
+    - name: slot_Shape_flag_no_pixels
       bit: 24
-      doc: Both weighted and unweighted moments were invalid
-    - name: slot_Shape_flag_unweighted
+      doc: no pixels to measure
+    - name: slot_Shape_flag_not_contained
       bit: 25
-      doc: Weighted moments converged to an invalid value; using unweighted moments
-    - name: slot_Shape_flag_shift
+      doc: center not contained in footprint bounding box
+    - name: slot_Shape_flag_parent_source
       bit: 26
-      doc: centroid shifted by more than the maximum allowed amount
-    - name: slot_Shape_flag_maxIter
+      doc: parent source, ignored
+    - name: slot_Shape_flag_galsim
       bit: 27
-      doc: Too many iterations in adaptive moments
-    - name: slot_Shape_flag_psf
-      bit: 28
-      doc: Failure in measuring PSF model shape
+      doc: GalSim failure
```

Comment:
Sorry, missed this the first time around -- why are we using GalSim in the AP pipeline in the first place?

Reply:
We're not; I just copied all of the bits verbatim from the HSM shape code. See lines 45-49 of https://github.com/lsst/meas_extensions_shapeHSM/blob/main/src/HsmShape.cc

Reply:
Then I suggest removing this one -- even if it gets produced in the DiaSource catalog, there's no reason to copy it here.

Reply:
I suppose we could only include mappings for the bits we're sure make sense, though it seemed easier to me at the time to just copy the entire thing for reasons of consistency.

Reply:
This was a few years back, but I think I remember Morrison saying that we didn't have many bits to spare. So it would be better to keep the flag map as compact as possible.